src/lib/locale/index.ts

```diff
@@ -29,6 +29,34 @@ export const routing = defineRouting({
       en: '/about',
       no: '/om-oss',
     },
+    '/about/leaderboard': {
+      en: '/leaderboard',
```

Comment:
these translations are wrong. you need to include the full path. /om-oss/ledertavle. Ledelsen is the wrong translation too this is an issue on every page